db/seeds.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'json'
```

Comment:
I don't see this being used in this file.

Reply:
resolved